src/v/cluster/health_monitor_backend.cc

```diff
@@ -572,6 +573,16 @@ ss::future<std::error_code> health_monitor_backend::collect_cluster_health() {
 
     auto old_reports = std::exchange(_reports, {});
 
+    // Collect cloud storage usage
+    cluster::cloud_storage_size_reducer reducer(
+      _topic_table,
+      _members,
+      _partition_leaders_table,
+      _connections,
+      topic_table_partition_generator::default_batch_size,
+      cloud_storage_size_reducer::default_retries_allowed);
+    _bytes_in_cloud_storage = co_await reducer.reduce();
```

Comment:
Adding this here means that we will perform this global reduce every 10 seconds. This is a pretty heavy operation as it potentially touches all shards on all nodes. As far as I can tell, `usage_manager` doesn't require this granularity. Do you think we could do something to call this less often?